src/Umbraco.Infrastructure/Services/Implement/PackagingService.cs

```diff
@@ -109,10 +131,14 @@ public class PackagingService : IPackagingService
     [Obsolete("Use DeleteCreatedPackageAsync instead. Scheduled for removal in Umbraco 15.")]
     public void DeleteCreatedPackage(int id, int userId = Constants.Security.SuperUserId)
     {
-        using ICoreScope scope = _coreScopeProvider.CreateCoreScope();
```

Comment:
This scope was disposed without being completed, so the `CreatedPackageSchemaRepository` will delete the file, but rollback the database changes!